app/controllers/application_controller.rb

```diff
@@ -1,9 +1,14 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_cart
+  helper_method :current_customer?
 
   def set_cart
     @cart = Cart.new(session[:cart])
   end
 
```

Comment:
Lets add another method called "current customer" which will do basically this.

Then edit this method so it returns true or false.